src/App.js

```diff
@@ -1,12 +1,176 @@
+    const [leeftijd, setLeeftijd] = useState(0);
+    const [postcode, setPostcode] = useState('');
+    const [bezorg, toggleBezorg] = useState('');
+    const [tijd, toggleTijd] = useState('');
+    const [message, setMessage] = useState('');
+    const [voorwaard, setVoorwaard] = useState('');
+
+    function resetBtn() {
+        setCountArdbei(0)
+        setCountBananen(0)
+        setCountAppels(0)
+        setCountKiwis(0)
+    }
+
+    function handleChange(e) {
+        e.preventDefault();
+        console.log(`
+    Voornaam: ${voornaam}, 
+    Achternaam: ${achternaam}, 
+    Leeftijd: ${leeftijd}, 
+    Postcode: ${postcode}, 
+    Bezorg: ${bezorg},
+    Tijd: ${tijd},
+    Opmerkingen: ${message},
+    Voorwaarden: ${voorwaard}
+    `);
+        console.log(`Aardbeiden: ${countArdbei}, Bananen: ${countBananen}, Appels: ${countAppels}, Kiwi's: ${countKiwis}`);
+    }
+    return (
+        <>
+            <section className="container">
+            <Counter
+                fruitName="🍓 Aardbeien"
+                type="button"
+                className="btn"
+                fruitValue={countArdbei}
+                increment={() => setCountArdbei(countArdbei => countArdbei + 1)}
+                decrement={() => setCountArdbei(countArdbei => countArdbei - 1)}
+                isDisabled={countArdbei === 0}
+            />
+            </section>
+            <section className="container">
+            <Counter
+                fruitName="🍌 Bananen"
+                type="button"
+                className="btn"
+                fruitValue={countBananen}
+                increment={() => setCountBananen(countBananen => countBananen + 1)}
+                decrement={() => setCountBananen(countBananen => countBananen - 1)}
+                isDisabled={countBananen === 0}
+            />
+            </section>
+            <section className="container">
+            <Counter
+                fruitName="🍏 Appels"
+                type="button"
+                className="btn"
+                fruitValue={countAppels}
+                increment={() => setCountAppels(countAppels => countAppels + 1)}
+                decrement={() => setCountAppels(countAppels => countAppels - 1)}
+                isDisabled={countAppels === 0}
+            />
+            </section>
+            <section className="container">
+            <Counter
+                fruitName="🥝 Kiwis"
+                type="button"
+                className="btn"
+                fruitValue={countKiwis}
+                increment={() => setCountKiwis(countKiwis => countKiwis + 1)}
+                decrement={() => setCountKiwis(countKiwis => countKiwis - 1)}
+                isDisabled={countKiwis === 0}
+            />
+            </section>
+            <Button type="button" className="btn_reset" onClick={resetBtn}>Reset</Button>
 
-export default App;
+            <form onSubmit={handleChange}>
+                <Formulier
```

Comment:
Naamgeving (waar mogelijk) in het Engels 😉

Reply:
ok zeker!!